invenio_administration/assets/semantic-ui/translations/invenio_administration/package.json

```diff
@@ -18,5 +18,8 @@
     "postextract_messages": "i18next-conv -l en -s ./messages/en/translations.json -t ./translations.pot",
     "compile_catalog": "node ./scripts/compileCatalog.js",
     "init_catalog": "node ./scripts/initCatalog"
+  },
+  "dependencies": {
+    "acorn": "^8.8.0"
```

Comment:
I don't think this is not needed, might be that while testing with different dependencies you didn't delete the `package-lock.json` and it was failing. Try to delete your local node_modules (if any) and also delete the `package-lock.json` and run `npm install` it should work correctly.

P.S. While adding new dependencies keep in mind that if it's added in `dependencies` it means that it's required in production, in this case it should be in `devDependencies`.

Reply:
The part that puzzles me is that, while acorn was not getting installed without fixing the dependency, other acorn related packages such as acorn-class-fields and acorn-jsx where getting installed. I tried deleting the node-modules and the package-lock, but basic acorn was still not getting installed.

Reply:
was that package needed? What it is needed for? 
If it is needed why it is not installed as devDependency ?